src/cbmc_starter_kit/template-for-repository/proofs/lib/summarize.py

```diff
@@ -97,23 +99,46 @@ def _get_status_and_proof_summaries(run_dict):
     return [statuses, proofs]
 
 
-def print_proof_results(out_file):
+def main(out_file):
     """
     Print 2 strings that summarize the proof results.
     When printing, each string will render as a GitHub flavored Markdown table.
     """
-    print("## Summary of CBMC proof results")
-    try:
-        with open(out_file, encoding='utf-8') as run_json:
-            run_dict = json.load(run_json)
-            summaries = _get_status_and_proof_summaries(
-                run_dict)
-            for summary in summaries:
-                print(_get_rendered_table(summary))
-    except Exception as ex: # pylint: disable=broad-except
-        logging.critical("Could not print results. Exception: %s", str(ex))
+    output = "## Summary of CBMC proof results\n\n"
+    with open(out_file, encoding='utf-8') as run_json:
+        run_dict = json.load(run_json)
+    status_table, proof_table = _get_status_and_proof_summaries(run_dict)
```

Comment:
Can you update `_get_status_and_proof_summaries()` such that the Litani pipeline called `print_tool_versions` isn't accounted for in either the status or proof tables?

https://github.com/ronakfof/mirror-s2n-tls/actions/runs/3906616093

is an example where this pipeline is incorrectly being accounted for as a proof.

Reply:
done, thanks